packages/experience-docs/source/components/PageWrapper/index.js

```diff
@@ -5,10 +5,12 @@ const PageWrapper = comp(({
     font
   }
 }) => ({
+  display: 'flex',
+  margin: '0 auto',
+  maxWidth: '50em',
```

Comment:
I think it would be better set max-width using `measure.wide` on the content section but leave `margin: auto` here. It should still center both sections together but they each have their own widths.